frontend/app/utils/auth-util.js

```diff
@@ -1,7 +1,13 @@
 class AuthUtil {
-  
+
   static showGuide(that, destination) {
     var div = document.getElementById(destination);
+    $(div).tooltip('show');
+    $(div).on('shown.bs.tooltip', function() {
+      setTimeout(function() {
+        $(div).tooltip('hide');
+      }, 1000);
+    });
```

Comment:
Ember.run.later((function() {
  //do something in here that will run in 2 seconds
}), 1000);